### PR TITLE
Ports xenobio borers & swarmers

### PR DESCRIPTION
**Pull request by ShadowDeath6**
title

also made the supr secret reaction chem less annoying to get

and before you say "oh shadow but you consistently got borers with the difficult reagent"

literally nobody else was doing it other than me and maybe X at one point

also removed colorful reagent because it doesn't fit with borers in any way

also fixed a meme bug where you could possess a dead borer

:cl: ShadowDeath6
rscadd: Cortical Borers have returned to xenobio. 
rscadd: Borers and swarmers can be gotten via a secret reaction with a gold slime extract.
rscadd: Borers can now create Rezadone in their hosts to heal cellular damage.
rscdel: Removed Colorful Reagent from borers.
tweak: Made the secret reaction chemical a bit less annoying to get.
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/recipes/slime_extracts.dm [hunk 1]
```diff
@@ -133,9 +133,28 @@
...
 	var/turf/T = get_turf(holder.my_atom)
-	T.visible_message("<span class='danger'>The slime extract begins to vibrate adorably !</span>")
+	T.visible_message("<span class='danger'>The slime extract begins to vibrate adorably!</span>")
 	addtimer(src, "chemical_mob_spawn", 50, FALSE, holder, 1, "Friendly Gold Slime", "neutral")
 
+/datum/chemical_reaction/slimecritstrange
+	name = "Slime Crit Strange"
+	id = "m_tele5"
+	result = null
+	required_reagents = list("omnizine" = 3) //Grind those donk pockets
```
Comment (Super3222) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm:143]: This should kept as strange reagent. The reason it's hard-to-get is because you are literally creating a considerable antagonist. That will spread, and will make more of itself, and will piss of tons of people. Sure, they don't have any reason to slaughter people at first but, once someone tries killing a borer the have plenty of reasons too. Which is what makes them a threat to the round, and invites chaos to the station. The normal reaction for security is to quarantine the hosts, and eliminate the borers. Borers aren't going to put up with that, they'll either kill all of security or any officer that seems them reproducing.

You get where I'm going with this, the excuse that they aren't harmful isn't something we can just go by. You've said it yourself, that these are no pAI type of borers. They mean serious business when it comes to reproducing.

The only way I can see this happening (where your goal is to make it easier to create borers, note I'm not saying change omnizine to something easier to grab, I'm saying if it's going to stay that way) is to make it so the new born borer CANNOT REPRODUCE and MUST SERVE THEIR MASTER. Otherwise, it should stay as strange reagent.

Because they ARE an antagonist, no matter how you put it. There's a /REASON/ why the xenomorphs you can hatch inside of xenobiology are /NOT/ the type that can reproduce and lack /MOST OF THE NORMAL XENOMORPH ABILITIES/. Domination. It's domination. That's the reason and the only thing it brings is chaos.

tl;dr this is a hugbox to borers

Comment (ShadowDeath6) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm:143]: changeling headslugs

at least it's not unstable mutagen, go bug botany or chef for your borers

Comment (Super3222) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm:143]: they are not antagonists that replicate themselves.

Comment (ShadowDeath6) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm:143]: they're more than capable of wiping a portion the station, which is more than borers can do, considering that borers literally have to keep their hosts alive or they can't do anything

Comment (ShadowDeath6) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm:143]: and be honest, do you honestly thing a xenobiologist in every round will bug chef / botany for omnizine every round? These aren't supposed to be locked behind 3 different jobs being willing to give you some of their shit.

Comment (Super3222) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm:143]: Robust xenobiologists will find out how to get it. It's a tough puzzle to crack and it's a secret you're adding to the game that'll probably be past by with only whispers (just like every other secret until someone becomes a buzzkill). You're literally making it easy to get almost as if it's just another normal feature to the game. Omnizine is not hard to get, donkpockets are super common with lavaland, and this shouldn't be easy to do in every single round. Remember that you got in trouble for pulling off shit like that.

**Comment by TheSadFox** [comment]
> -"secret" reaction
> -litteraly anyone can look up the fucking code

**Comment by ShadowDeath6** [comment]
it's probably going to be a bit less secret than colorful reagent which people don't spam too often

File: code/modules/reagents/chemistry/recipes/slime_extracts.dm (hunk 1 above)
Comment (ghost) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm:143]: @shadowdeath6

Xenobio-made ling headslugs are nonantags and have to obey their master's orders, same as anything. That includes the actual changeling, or if a scientist makes themselves a changeling. Unless given objectives they're a nonantag.

Comment (ShadowDeath6) [inline code/modules/reagents/chemistry/recipes/slime_extracts.dm:143]: Making this more annoying to get just discourages people from utilizing a feature. I was thinking about making it earthsblood since that's a chem from one other department rather than 2.

File: code/modules/mob/living/simple_animal/borer/borer.dm [hunk 2]
```diff
@@ -49,14 +49,17 @@ var/total_borer_hosts_needed = 10
 	borer_chems += /datum/borer_chem/perfluorodecalin
 	borer_chems += /datum/borer_chem/spacedrugs
 	borer_chems += /datum/borer_chem/mutadone
-	borer_chems += /datum/borer_chem/creagent
+	//borer_chems += /datum/borer_chem/creagent
```
Comment (oisin100) [inline code/modules/mob/living/simple_animal/borer/borer.dm:52]: Nope.

Comment (ShadowDeath6) [inline code/modules/mob/living/simple_animal/borer/borer.dm:52]: provide a reason lore wise why it's necessary and I'll add it back in

Comment (oisin100) [inline code/modules/mob/living/simple_animal/borer/borer.dm:52]: Lore:
"Oisin liked creagent, so when the almighty headcoderbus god decided to create borers in such a way that they were gifted with the power of creagent"

Comment (ShadowDeath6) [inline code/modules/mob/living/simple_animal/borer/borer.dm:52]: it's supposed to be hard to get for a reason, it's like the endgame meme for chem, not random thing in borers

**Comment by ShadowDeath6** [comment]
FUCK IT DIDN"T WORK

**Comment by ShadowDeath6** [comment]
:^)
